pgx_lwt_mirage/src/pgx_lwt_mirage.mli

```diff
@@ -0,0 +1,28 @@
+(* Draft for cherry-picking into pgx.
```

Comment:
Weird comment?

Reply:
I think this came from when i copied the license header from the gist linked to in the issue for mirage. I wasn't sure what content i'm allowed to touch in such situation

Reply:
We can edit this comment as long as we leave the license block alone and only add to the copyright section.

Reply:
Updated the header to remove the comment about cherry-picking